lib/JSON/Validator/OpenAPI/Dancer2.pm

```diff
@@ -0,0 +1,72 @@
+package JSON::Validator::OpenAPI::Dancer2;
+
+use Carp qw(confess);
+use Hash::MultiValue;
```

Comment:
Settled in IRC: As it's already a dependency of `Dancer2` we won't depend on this ourself.